sections/02-introduction.qmd

```diff
@@ -48,9 +48,9 @@ by others, including NumFOCUS, the R Consortium, the Python Software
 Foundation, DuckDB, Pandoc, pyodide, and ProseMirror, as well as dozens
 of smaller projects via the Open Source Collective or directly on
 Github. Additional information about our products and company
-contributions for the past two years can be found in our ["Year In
+contributions can be found in our ["Year In
 Review" blog
-posts](https://posit.co/blog/2023-posit-year-in-review/).
+posts](https://posit.co/blog/?post_tag=year-in-review).
```

Comment:
@cp362 Got the tag! How does this look? I also dropped the "past two years", because now this applies to all years (and I suspect the _past two_ was because we hadn't done a report in two years last time).